ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_p2p_links.schema.yml

```diff
@@ -105,10 +105,13 @@ $defs:
           type: bool
           default: false
           description: Enable BFD (only considered for BGP).
-        ptp_enable:
-          type: bool
-          default: false
-          description: Enable PTP
+        ptp:
```

Comment:
I think it would make sense to be able to set/override the profile here since it is ad-hoc p2p links (separate PR)

Reply:
nevermind. There is a separate issue on this and it should be fixed in a separate PR.